ichnaea/locate/provider.py

```diff
@@ -600,11 +610,11 @@ def _prepare_data(self, data):
             if wifi_query:
                 wifi_queries.append(wifi_query)
 
-        query = {}
-        if cell_queries:
-            query['cellTowers'] = cell_queries
-        if wifi_queries:
-            query['wifiAccessPoints'] = wifi_queries
+        query = {
+            'cellTowers': cell_queries,
+            'wifiAccessPoints': wifi_queries,
+            'fallbacks': data.get('fallbacks', {}),
+        }
```

Comment:
I still prefer to not send empty arrays. If an inbound query didn't contain any cell data, we can just avoid sending `"cellTowers": []` in the outbound query, same for wifi or fallbacks.